src/plugins/custom_integrations/server/language_clients/index.ts

```diff
@@ -0,0 +1,181 @@
+    title: i18n.translate('customIntegrations.languageclients.JavascriptTitle', {
+      defaultMessage: 'Elasticsearch JavaScript Client',
+    }),
+    icon: 'nodejs.svg',
+    description: i18n.translate('customIntegrations.languageclients.JavascriptDescription', {
+      defaultMessage:
+        'Start building your custom application on top of Elasticsearch with the official Node.js client.',
+    }),
+    docUrlTemplate: `${ELASTICSEARCH_CLIENT_URL}/javascript-api/{branch}/introduction.html`,
+  },
+  {
+    id: 'ruby',
+    title: i18n.translate('customIntegrations.languageclients.RubyTitle', {
+      defaultMessage: 'Elasticsearch Ruby Client',
+    }),
+    icon: 'ruby.svg',
+    description: i18n.translate('customIntegrations.languageclients.RubyDescription', {
+      defaultMessage:
+        'Start building your custom application on top of Elasticsearch with the official Ruby client.',
+    }),
+    docUrlTemplate: `${ELASTICSEARCH_CLIENT_URL}/ruby-api/{branch}/ruby_client.html`,
+  },
+  {
+    id: 'go',
+    title: i18n.translate('customIntegrations.languageclients.GoTitle', {
+      defaultMessage: 'Elasticsearch Go Client',
+    }),
+    icon: 'go.svg',
+    description: i18n.translate('customIntegrations.languageclients.GoDescription', {
+      defaultMessage:
+        'Start building your custom application on top of Elasticsearch with the official Go client.',
+    }),
+    docUrlTemplate: `${ELASTICSEARCH_CLIENT_URL}/go-api/{branch}/overview.html`,
+  },
+  {
+    id: 'dotnet',
+    title: i18n.translate('customIntegrations.languageclients.DotNetTitle', {
+      defaultMessage: 'Elasticsearch .NET Client',
+    }),
+    icon: 'dotnet.svg',
+    description: i18n.translate('customIntegrations.languageclients.DotNetDescription', {
+      defaultMessage:
+        'Start building your custom application on top of Elasticsearch with the official .NET client.',
+    }),
+    docUrlTemplate: `${ELASTICSEARCH_CLIENT_URL}/net-api/{branch}/index.html`,
+  },
+  {
+    id: 'php',
+    title: i18n.translate('customIntegrations.languageclients.PhpTitle', {
+      defaultMessage: 'Elasticsearch PHP Client',
+    }),
+    icon: 'php.svg',
+    description: i18n.translate('customIntegrations.languageclients.PhpDescription', {
+      defaultMessage:
+        'Start building your custom application on top of Elasticsearch with the official .PHP client.',
+    }),
+    docUrlTemplate: `${ELASTICSEARCH_CLIENT_URL}/php-api/{branch}/index.html`,
+  },
+  {
+    id: 'perl',
+    title: i18n.translate('customIntegrations.languageclients.PerlTitle', {
+      defaultMessage: 'Elasticsearch Perl Client',
+    }),
+    icon: 'perl.svg',
+    description: i18n.translate('customIntegrations.languageclients.PerlDescription', {
+      defaultMessage:
+        'Start building your custom application on top of Elasticsearch with the official Perl client.',
+    }),
+    docUrlTemplate: `${ELASTICSEARCH_CLIENT_URL}/perl-api/{branch}/index.html`,
```

Comment:
Per test failures in https://github.com/elastic/kibana/pull/114047, there is no 7.x version of this doc. I suggest using "current" or "master" instead in this case:
```suggestion
    docUrlTemplate: `${ELASTICSEARCH_CLIENT_URL}/perl-api/current/index.html`,
```